### DISCUSSION TITLE
プルリクエストのレビューを最優先にするというルール化

### DISCUSSION THREAD

**Discussion by riscait** [altive/handbook#27] [Polls]
> [Xのポスト](https://twitter.com/koh110/status/1731625310641344839)や[記事](https://zenn.dev/team_soda/articles/4faec057f57e74)で「PRのレビューを最優先にしたらチームの生産性が上がった」というのをみて自分も共感するところがあったので、合意が取れればその運用もルール化してみたい✍️

**Comment by naipaka** [comment]
やってみて課題出てくる場合もあると思うので一回ルール化して試してみても良さそうという意味で投票しました🗳️

**Comment by riscait** [comment]
コードレビューの最優先化するのであれば、レビュー件数（担当数）の調整を行わないと、担当が多い人は（極端に言うと）自分の仕事が全く進められなくなる事態に陥る可能性もあるかも💭

**Comment by riscait** [comment]
最優先のルール化する。 Issue化 (#30 )
